pallet/src/benchmarking.rs

```diff
@@ -1,46 +1,40 @@
 //! Benchmarking setup for pallet-move.
 
-use frame_benchmarking::*;
+use frame_benchmarking::benchmarks;
+#[cfg(test)]
+use frame_benchmarking::impl_benchmark_test_suite;
 use frame_system::{Config as SysConfig, RawOrigin};
-use move_core_types::account_address::AccountAddress;
 use move_vm_backend::types::MAX_GAS_AMOUNT;
-pub use sp_core::{crypto::Ss58Codec, sr25519::Public};
-use sp_std::{vec, vec::Vec};
+use sp_core::crypto::Ss58Codec;
 
-#[cfg(test)]
-use crate::mock::*;
-use crate::{balance::BalanceOf, *};
+use crate::{balance::BalanceOf, mock_utils::*, *};
 
-const LIMIT: u128 = 30_000_000_000_000;
-const BOB_ADDR: &str = "5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty";
-const ALICE_ADDR: &str = "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY";
-const DAVE_ADDR: &str = "5DAAnrj7VHTznn2AWBemMuyBwZWs6FNFjdyVXUeYum3PTXFy";
-const EVE_ADDR: &str = "5HGjWAeFDfFCWPsjFQdVV2Msvz2XtMktvgocEZcCj68kUMaw";
+const LIMIT: u128 = 60_000_000_000_000;
 
 benchmarks! {
     where_clause { where
         T: Config + SysConfig,
-        T::AccountId: From<Public>,
+        T::AccountId: Ss58Codec,
     }
 
     execute {
         let n in 0 .. 7;
 
-        let (bob_32, bob_mv) = account_address::<T>(BOB_ADDR);
-        let (alice_32, alice_mv) = account_address::<T>(ALICE_ADDR);
-        let (dave_32, dave_mv) = account_address::<T>(DAVE_ADDR);
-        let (eve_32, eve_mv) = account_address::<T>(EVE_ADDR);
+        let (bob_32, bob_mv) = account_n_address::<T>(BOB_ADDR);
+        let (alice_32, alice_mv) = account_n_address::<T>(ALICE_ADDR);
+        let (dave_32, dave_mv) = account_n_address::<T>(DAVE_ADDR);
+        let (eve_32, eve_mv) = account_n_address::<T>(EVE_ADDR);
 
         // Our benchmark plan (each is a test scenario with different parameters).
         let script_bcs = [
-            car_wash_initial_coin_miniting(&bob_mv),
-            multiple_signers_init_module(&bob_mv),
-            car_wash_register_new_user(&alice_mv),
-            car_wash_buy_coin(&alice_mv, 1),
-            car_wash_wash_car(&alice_mv),
-            multiple_signers_rent_apartment(&alice_mv, &dave_mv, &eve_mv, 1),
-            multiple_signers_rent_apartment(&alice_mv, &dave_mv, &eve_mv, 1),
-            multiple_signers_rent_apartment(&alice_mv, &dave_mv, &eve_mv, 1),
+            car_wash_initial_coin_miniting().to_vec(),
+            multiple_signers_init_module().to_vec(),
+            car_wash_register_new_user().to_vec(),
```

Comment:
Where are those users gone? Isn't this supposed to be a transaction script here? 🤔

Reply:
That is a long story... I switched from bytecode scripts and generating the transaction scripts to load the out-of-the-box transaction scripts. The main reason is that I was trying to move the code in `benchmark_only` also to `mock_utils`, but this seems not to be possible 😩